spec/system/admin/bulk_order_management_spec.rb

```diff
@@ -476,37 +476,30 @@
         it "allows filters to be used in combination" do
           expect(page).to have_selector "tr#li_#{li1.id}"
           expect(page).to have_selector "tr#li_#{li2.id}"
-          find("#s2id_order_cycle_filter").click
-          find(".select2-result-label", text: oc1.name.to_s).click
+          click_on_select2 oc1.name, from: "order_cycle_filter"
           expect(page).to have_selector "tr#li_#{li1.id}"
           expect(page).to have_no_selector "tr#li_#{li2.id}"
-          find("#s2id_distributor_filter").click
-          find(".select2-result-label", text: d1.name.to_s).click
-          find("#s2id_supplier_filter").click
-          find(".select2-result-label", text: s1.name.to_s).click
+          click_on_select2 d1.name, from: "distributor_filter"
+          click_on_select2 s1.name, from: "supplier_filter"
           expect(page).to have_selector "tr#li_#{li1.id}"
           expect(page).to have_no_selector "tr#li_#{li2.id}"
           find("#s2id_distributor_filter").click
           find(".select2-result-label", text: d2.name.to_s).click
-          find("#s2id_supplier_filter").click
-          find(".select2-result-label", text: s2.name.to_s).click
+          click_on_select2 d2.name, from: "distributor_filter"
```

Comment:
We probably should remove the two previous lines right? 

```
find("#s2id_distributor_filter").click
find(".select2-result-label", text: d2.name.to_s).click
```

Reply:
Argh missed those. Fixing.

Reply:
Done, thank you!